### ISSUE TITLE
run Transgan with single gpu

### ISSUE THREAD

**Issue by Stephenlove** [VITA-Group/TransGAN#48]
hello,your work is very exciting but i need to modify your code to run successfully on a GPU have tried many times,but i have not succeded.May i ask your advice?

**Comment by yueruchen** [comment]
I feel it would be a great challenge to run it on single gpu. Our smallest experiments takes 1.5 x 8 gpu days on CIFAR-10 dataset. If you insist on doing this, I would suggest you to fully use the advantage of gradient checkpointing (our codebase supports this feature) and use minimum batchsize as you can.

**Comment by Jamie-Cheung** [comment]
> I feel it would be a great challenge to run it on single gpu. Our smallest experiments takes 1.5 x 8 gpu days on CIFAR-10 dataset. If you insist on doing this, I would suggest you to fully use the advantage of gradient accumulation (our codebase supports this feature) and use minimum batchsize as you can.

can you tell me  what is your gpu memory?

**Comment by yueruchen** [comment]
@Jamie-Cheung 
I dont remember clearly but it takes more than 11gb if you use only 2 gpus
